articles/azure-app-configuration/faq.yml

```diff
@@ -146,15 +141,16 @@ sections:
 
       - question: Why can't I create an App Configuration store with the same name as one that I just deleted?
         answer: |
-          When a standard tier App Configuration store is deleted the name is reserved for seven days after deletion. It is possible to recreate a store with the same name before the reservation period has elapsed but it must be in the same subscription, resource group, and region as the original store. If the resource group that contained the store was deleted it will need to be recreated in the same subscription first before recreating the store inside of it.
-
-          The App Configuration store cannot be created in a different subscription/resource group until the retention period has elapsed. However, the resource can be recreated in the original subscription/resource group and then moved to a new subscription/resource group by following [these instructions](../azure-resource-manager/management/move-resource-group-and-subscription.md#use-the-portal).
+          All App Configuration stores in the standard tier have automatically enabled the [soft-delete](./concept-soft-delete) feature. When a standard tier App Configuration store is deleted, its name is reserved for the retention period. To recreate a store with the same name before the retention period expires, you need to [purge the soft-deleted store](./howto-recover-deleted-stores-in-azure-app-configuration#list-recover-or-purge-a-soft-deleted-app-configuration-store) first, provided the store doesn't have purge protection enabled.
 
-          Recreating an App Configuration store during the retention period is currently only supported by the Azure CLI using the `az appconfig create` [command](/cli/azure/appconfig#az_appconfig_create).
+          App Configuration previously offered an option to overwrite a soft-deleted store as instructed below. This option will be retired by April 30, 2022. Migrate to use the purge function or set a shorter retention period if you often need to recreate a store with the same name.
```

Comment:
Shouldn't it be "deleted store" instead of "soft-deleted" store? For the user soft deleted wasn't really a concept/feature earlier.

Reply:
"This option will be retired on/after/on or after April 30,2022"

Reply:
We have the term "soft deleted store" throughout our concept doc
https://docs.microsoft.com/en-us/azure/azure-app-configuration/concept-soft-delete

BTW, this paragraph won't live long. I will delete it altogether after 4/30.

Reply:
Good suggestion. Changed to "**on** April 30, 2022".

Reply:
> We have the term "soft deleted store" throughout our concept doc https://docs.microsoft.com/en-us/azure/azure-app-configuration/concept-soft-delete
> 
> BTW, this paragraph won't live long. I will delete it altogether after 4/30.

Yep, this will be gone in a month and is alright. I commented as this one is specifically calling out soft-deleted stores from the time before soft delete was introduced. Concept doc's job is to introduce the user to the soft-deleted state.

Reply:
Technically speaking, the "soft delete" is not a new concept. We have introduced it since last September already. What is new is the "recover" and "purge" functions that users can do by themselves now (instead of calling Microsoft Support).